zerver/views/auth.py

```diff
@@ -822,13 +823,13 @@ def dev_direct_login(request: HttpRequest, **kwargs: Any) -> HttpResponse:
     if (not dev_auth_enabled()) or settings.PRODUCTION:
         # This check is probably not required, since authenticate would fail without
         # an enabled DevAuthBackend.
-        return HttpResponseRedirect(reverse('dev_not_supported'))
+        return HttpResponseRedirect(reverse('config_error', kwargs={'error_category_name': 'dev'}))
     email = request.POST['direct_email']
     subdomain = get_subdomain(request)
     realm = get_realm(subdomain)
     user_profile = authenticate(dev_auth_username=email, realm=realm)
     if user_profile is None:
-        return HttpResponseRedirect(reverse('dev_not_supported'))
+        return HttpResponseRedirect(reverse('config_error', kwargs={'error_category_name': 'dev'}))
```

Comment:
Can we change the HttpResponseRedirect(...) in this file to use `redirect_to_config_error()` instead? That's what the function is meant for, so it seems better to be consistent and use for returning config error redirects.

Reply:
Okay, Changed!